src/lib/ngsi/ContextAttribute.cpp

```diff
@@ -351,6 +351,108 @@ std::string ContextAttribute::render
 
 /* ****************************************************************************
 *
+* renderV2 - 
+*/
+std::string ContextAttribute::renderV2
+(
+  ConnectionInfo*     ciP,
+  RequestType         request,
+  bool                comma
+)
+{
+  std::string  out;
+  bool         isNumber = false;
+
+  if (type == "number")
+  {
+    isNumber = true;
+  }
+
+  if (((type == "") || (isNumber == true)) && (metadataVector.size() == 0))
+  {
+    if (isNumber == true)
+    {
+      out = "\"" + name + "\":" + value;
+    }
+    else
+    {
+      if (compoundValueP == NULL)
+      {
+        out = "\"" + name + "\":\"" + value + "\"";
+      }
+      else
+      {
+        if (compoundValueP->isObject())
+        {
+          out = "\"" + name + "\":{" + compoundValueP->renderV2() + "}";
+        }
+        else if (compoundValueP->isVector())
+        {
+          out = "\"" + name + "\":[" + compoundValueP->renderV2() + "]";
+        }
+        else
+        {
+          out = "\"" + name + "\":\"" + compoundValueP->renderV2() + "\"";  // Can toplevel be a String?
+        }
+      }
+    }
+  }
+  else
+  {
+    out = "\"" + name + "\":{";
+
+    if (isNumber == true)
+    {
+      out += "\"value\":" + value;
+    }
+    else
+    {
+      if (compoundValueP == NULL)
+      {
+        out += "\"value\":\"" + value + "\"";
+      }
+      else
+      {
+        if (compoundValueP->isObject())
+        {
+          out = "\"" + name + "\":{" + compoundValueP->renderV2() + "}";
+        }
+        else if (compoundValueP->isVector())
+        {
+          out = "\"" + name + "\":[" + compoundValueP->renderV2() + "]";
+        }
+        else
+        {
+          out = "\"" + name + "\":\"" + compoundValueP->renderV2() + "\"";  // Can toplevel be a String?
+        }
+      }
+    }
+
+    if ((type != "") && (isNumber == false))
+    {
+      out += ",\"type\":\"" + type + "\"";
+    }
+
+    if (metadataVector.size() > 0)
+    {
+      out += "," + metadataVector.renderV2(ciP, request, false);
+    }
+
+    out += "}";
+  }
+
+  if (comma)
```

Comment:
isLastElement is used finally.
Commit: https://github.com/telefonicaid/fiware-orion/commit/0bc1f0199f2bc6055a2c24ae3ab24a71a422674c